### ISSUE TITLE
Build is broken

### ISSUE THREAD

**Issue by mykelk** [JuliaPOMDP/POMDPs.jl#469]
The documentation is failing to build as of a couple weeks ago and we have a failed badge. Checking the log, we have:
```
ERROR: LoadError: UndefVarError: @POMDP_require not defined
```

**Comment by zsunberg** [comment]
This is because we were about to release version 1.0, but then I got married, so that was distracting. Should be resolved in a week.

**Comment by zsunberg** [comment]
Build is passing, but still haven't released 1.0
